web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -55,6 +55,34 @@ export class DataStoreService {
       );
   }
 
+  /**
+   * Returns an Observable that loads and emits the projects to which the user
+   * is part of.
+   *
+   */
+  loadAccessibleProject$(userEmail: string): Observable<List<Project>> {
+    return this.db
+      .collection('projects')
+      .snapshotChanges()
+      .pipe(
+        map(actions =>
+          List(
+            actions
+              .filter(a => {
```

Comment:
Our Firebase rules prevent users from fetching projects they cannot access, so you can't fetch _all projects_ and then filter them the client side. Instead, you'll need to use a query like the Android client does to filter:

https://github.com/google/ground-android/blob/cfa10ebd851cfc18e6bbcd14483cc5b5a4f4f709/gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/ProjectsCollectionReference.java#L46-L51